inetum-realjobs-backend/src/main/java/world/inetum/realdolmen/realjobs/services/ResumeService.java

```diff
@@ -0,0 +1,166 @@
+        logger.info("User {} removed education {}", currentUserJs.getId(), id);
+        return resume.getEducationList();
+    }
+
+    public List<Education> getEducationList() {
+        JobSeeker currentUserJs = securityService.getJobSeeker();
+        logger.info("User {} fetched their educationList", currentUserJs.getId());
+        return currentUserJs.getResume().getEducationList();
+    }
+
+    public List<Experience> addExperience(Experience newExperience) {
+        JobSeeker currentUserJs = securityService.getJobSeeker();
+        Resume resume = currentUserJs.getResume();
+        resume.addExperience(newExperience);
+        currentUserJs.setResume(resume);
+        accountRepository.save(currentUserJs);
+        logger.info("User {} added an experience {}", currentUserJs.getId(), newExperience);
+        return resume.getExperienceList();
+    }
+
+    public List<Experience> removeExperience(long id) {
+        JobSeeker currentUserJs = securityService.getJobSeeker();
+        Resume resume = currentUserJs.getResume();
+        resume.getExperienceList().removeIf(i -> i.getId().equals(id));
+        currentUserJs.setResume(resume);
+        accountRepository.save(currentUserJs);
+        logger.info("User {} removed experience {}", currentUserJs.getId(), id);
+        return resume.getExperienceList();
+    }
+
+    public List<Experience> getExperienceList() {
+        JobSeeker currentUserJs = securityService.getJobSeeker();
+        logger.info("User {} fetched their experienceList", currentUserJs.getId());
+        return currentUserJs.getResume().getExperienceList();
+    }
+
+    public String setSummary(String newSummary) {
+        JobSeeker currentUserJs = securityService.getJobSeeker();
+        Resume resume = currentUserJs.getResume();
+        resume.setSummary(newSummary);
+        currentUserJs.setResume(resume);
+        accountRepository.save(currentUserJs);
+        logger.info("User {} set their summary", currentUserJs.getId());
+        return resume.getSummary();
+    }
+
+    public String getSummary() {
+        JobSeeker currentUserJs = securityService.getJobSeeker();
+        logger.info("User {} fetched their summary", currentUserJs.getId());
+        return currentUserJs.getResume().getSummary();
+    }
+
+    public ResumeStatus setStatus(ResumeStatus newStatus) {
+        JobSeeker currentUserJs = securityService.getJobSeeker();
+        Resume resume = currentUserJs.getResume();
+        resume.setStatus(newStatus);
+        currentUserJs.setResume(resume);
+        accountRepository.save(currentUserJs);
+        logger.info("User {} set their status", currentUserJs.getId());
+        return resume.getStatus();
+    }
+
+    public ResumeStatus getStatus() {
+        JobSeeker currentUserJs = securityService.getJobSeeker();
+        logger.info("User {} fetched their status", currentUserJs.getId());
+        return currentUserJs.getResume().getStatus();
+    }
+
+    public Account getAccount() {
```

Comment:
Why not return a `JobSeeker`?

Reply:
All required data was available in Account, didn't think it was necessary.